Sources/VLCLibraryViewController.m

```diff
@@ -406,8 +406,8 @@ - (void)_displayEmptyLibraryViewIfNeeded
 
     if ([_mediaDataSource numberOfFiles] == 0) {
         _inFolder = (_libraryMode == VLCLibraryModeFolder || _libraryMode == VLCLibraryModeCreateFolder);
```

Comment:
the VLCLibrarycontroller doesn't need to be touched here. It's gonna be deleted anyway